packages/expo-core/android/src/main/java/expo/core/utilieties/FileUtilities.java

```diff
@@ -1,13 +1,12 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
-package expo.modules.print;
+package expo.core.utilieties;
```

Comment:
Let's not add this file to `expo-core`

Reply:
+ spelling

Reply:
I would rather add it here (or some corresponding one in `expo-core`), because more than one module already share this logic and here I need to use it again (that would be at least 3rd module), so here's my point.